### ISSUE TITLE
Confusion about this program

### ISSUE THREAD

**Issue by ning-mz** [SCNU203/GeoQA-Plus#2]
Hi, thanks for sharing your DPE-NGS. But the program is not working and something in code looks strange.

1. This repo missing many things to run.
2. GeoQA is a Chinese-based dataset but RoBERTa-base not support Chinese, in the paper you did not specified which RoBERTa model is used and there is just a path "model_name = "data/pretrain/Roberta" in your code, I'm not sure which RoBERTa is used. Is it "[chinese-roberta-wwm-ext](https://huggingface.co/hfl/chinese-roberta-wwm-ext)"?
3. RoBERTa use a pretrained vocab, but in your code it looks you pass the token_ids by using a new tokenizer created by allennlp, is it appropriate that use a different vocab on pretrained model.
4. When using DPE encoder, it requires both tokens and mask (def forward(self, src, source_mask)), but in NGS_Aux, there is just "state = self._encode(source_tokens)", which missing source_mask.
5. The ResNet in code is under "with torch.nograd()", but in the paper and config.json file, it has a learning rate 1e-5, I'm not sure about this.

I just started to pay attention to GMWPs, looking for your response, many thanks.
